docs/source/changelog.md

```diff
@@ -4,9 +4,49 @@
 
 ## [Unreleased]
 
+## 3.0
+
+### [3.0.0] - 2022-03-10
+
+This release is entirely about replacing a k8s api-server client library
+KubeSpawner has relied on that required us to use multiple threads. It is being
+replaced with one that enabled us to rely on `async`/`await` and an associated
+[event loop](https://docs.python.org/3/library/asyncio-eventloop.html).
+
+A key motivation for this change was that it reduced the complexity of the
+project's code, but it has also been found to reduce transient issues within the
+k8s cluster. For more technical details about this, see
+[#563](https://github.com/jupyterhub/kubespawner/pull/563).
```

Comment:
I think these two paragraphs are good to put in a release announcement.

I'm bit worried about a large block of text in this ever growing changelog though- we don't have seperate release pages. Do you reckon it's fine anyway?

Reply:
It would be an improvement to refine the changelog structure, but I don't want to consider that as part of trying to get an individual release out =/

I think that if this is relevant enough to say, it should stay in the main source of knowledge rather than for example be written on discourse or as a GitHub Release.

Reply:
I've removed these paragraphs! With your briefer summary they felt a bit redundant and less felt like more.